lib/training/hf_trainer.py

```diff
@@ -37,6 +44,16 @@ def _wrap_model(self, model, training=True):
         return IgnoreGradManipulations(super()._wrap_model(model, training=training),
                                        override_zero_grad=self.collaborative_optimizer.grad_averager.reuse_grad_buffers)
 
+    def get_country_info(self):
+        # As this method is only a nice to have, if ever the command fails for any reason we move on to something else
+        try:
+            response = urllib.request.urlopen(URL_IP_INFO)
+            data = json.load(response)
+            country = data["country"]
+        except Exception:
+            country = ""
+        return country
+
```

Comment:
I know that bare except is not a good practice, nevertheless I thought here that it was useless to risk to have an error at this level for this information which is only a "nice to have"